tcl/response.tcl

```diff
@@ -405,11 +425,50 @@ proc qc::response2html_snippet {} {
                            [h div class message [join $message_elements \n]] \
                            [h div class record [join $record_elements \n]] \
                            [h div class action [join $action_elements \n]] \
+                           [h div class data $data_html] \
                            [h div class extended [join $extended_elements \n]]
                           ] \n] \
                ]
 }
 
+proc qc::response_tson2html_fragment {tson} {
```

Comment:
It's not returning a full HTML document, and the tson being passed in isn't necessarily the full data.data property.